app/models/team.rb

```diff
@@ -4,10 +4,14 @@
 #
 #  id         :integer          not null, primary key
 #  name       :string
+#  description:text
+#  kindofsport:string
+#  private    :boolean
 #  created_at :datetime         not null
 #  updated_at :datetime         not null
 #
 
 class Team < ApplicationRecord
   validates :name, presence: true
+  validates :private, inclusion: { :in => [true, false] }
```

Comment:
According to a comment to https://stackoverflow.com/a/4721574, you can use the following shorter form instead:

```ruby
validates :private, inclusion: [true, false]
```